src/constants/translations/en/guest-home-page.json

```diff
@@ -23,7 +23,7 @@
   },
   "whatCanYouDo": {
     "title": "What can you do",
-    "description": "Whether you’re looking for a tutor or to looking become one – you’ve come to the right place.",
+    "description": "Whether you’re looking for a tutor or looking become one – you’ve come to the right place.",
```

Comment:
This description doesn't match with the requirement:
<img width="615" alt="Screenshot 2024-04-24 at 22 58 29" src="https://github.com/Space2Study-UA-1156/Client-02/assets/28622400/3c4efa0c-9109-4c17-bb24-171f6763517f">

Reply:
Done!
May I ask you check again? :)